File: SDKs/bungie.net/2.11.1/go/pkg/models/shared/searchresultoftrendingentry.go
package shared

type SearchResultOfTrendingEntry struct {
	HasMore                      *bool
	Query                        *QueriesPagedQuery
	ReplacementContinuationToken *string
	Results                      []TrendingTrendingEntry
	TotalResults                 *int32
	UseTotalResults              *bool
}

File: SDKs/bungie.net/2.11.1/typescript/src/sdk/models/shared/searchresultoftrendingentry.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { QueriesPagedQuery } from "./queriespagedquery";
import { TrendingTrendingEntry } from "./trendingtrendingentry";



export class SearchResultOfTrendingEntry extends SpeakeasyBase {
  @SpeakeasyMetadata()
  hasMore?: boolean;

  @SpeakeasyMetadata()
  query?: QueriesPagedQuery;

  @SpeakeasyMetadata()
  replacementContinuationToken?: string;

  @SpeakeasyMetadata({ elemType: TrendingTrendingEntry })
  results?: TrendingTrendingEntry[];

  @SpeakeasyMetadata()
  totalResults?: number;

  @SpeakeasyMetadata()
  useTotalResults?: boolean;
}
